packages/web/src/services/solana/WithdrawUSDC.ts

```diff
@@ -31,80 +40,163 @@ export const getFundDestinationTokenAccountFees = async (
 }
 
 /**
- * Creates instructions to swap USDC from a user bank into
- * SOL, which is deposited into the user's root solana account.
+ * Creates instructions to swap from a user bank token into the given wallet as SOL.
+ * These instructions are allowed in relay because every created token account is closed.
  */
-export const getSwapUSDCUserBankInstructions = async ({
-  amount,
+export const createSwapUserbankToSolInstructions = async ({
+  mint,
+  outSolAmount,
+  wallet,
```

Comment:
nit: thought we were trying to move away from wallet as a name for things

Reply:
I'm starting to see the use of it as a differentiator from a generic "account" and the actual "wallet" the user is aware of. I think most other things are moving _towards_ using "wallet" nomenclature in Solana-ville, actually, so maybe we should too?

Though I get the confusion with our existing ERC wallets... thoughts?